src/errors.ts

```diff
@@ -131,10 +131,16 @@ export enum ValidationErrorCodes {
     InvalidOrder = 1007,
     InternalError = 1008,
     TokenNotSupported = 1009,
+    FieldInvalid = 1010,
```

Comment:
![image](https://user-images.githubusercontent.com/210914/88566658-30ba4780-cfeb-11ea-81db-f95b8c67a8ad.png)
We should update the documentation here

Reply:
https://github.com/0xProject/website/pull/318